packages/cli/package.json

```diff
@@ -32,8 +32,7 @@
     "prebuild": "rimraf dist",
     "build": "tsc",
     "test": "mocha --timeout 60000 -r ts-node/register 'src/**/**.test.ts'",
-    "coverage": "nyc npm run test",
-    "prepare": "npm run build"
```

Comment:
Why is the `prepare` script being removed? IIRC we rely on it to transpile before publishing to npm.